DataAccessLayer/Services/EmployeeService.cs

```diff
@@ -28,5 +31,28 @@ public Employee GetEmployee (string email, string password)
 
             return employee;
         }
+
+        /// <summary>
+        /// This method creates a new Employee record with default values and returns it. 
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="email"></param>
+        /// <param name="password"></param>
+        /// <param name="roleId"></param>
+        /// <returns></returns>
+        public Employee PostEmployee(string name, string email, string password, int roleId)
```

Comment:
I would rename this to CreateEmployee or SaveEmployee
When we talk about Post, it refers to a HTTP Method that belongs to the presentation layer, so it could cause confuse